components/ActionSheet/examples/ActionSheetSwiftExample.swift

```diff
@@ -0,0 +1,201 @@
+    showActionSheet(data[indexPath.row].type)
+  }
+}
+
+extension ActionSheetSwiftExample : UITableViewDataSource {
+  func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+    let cell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier, for: indexPath)
+    cell.textLabel?.text = data[indexPath.row].label
+    return cell
+  }
+
+  func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    return data.count
+  }
+}
+
+extension ActionSheetSwiftExample {
+  static var actionOne: MDCActionSheetAction {
+    return MDCActionSheetAction(title: "Home",
+                                image: UIImage(named: "Home")!) { (_) in
+                                  print("Home action") }
+  }
+
+  static var actionTwo: MDCActionSheetAction {
+    return MDCActionSheetAction(title: "Favorite",
+                                image: UIImage(named: "Favorite")!) { (_) in
+                                  print("Favorite action") }
+  }
+
+  static var actionThree: MDCActionSheetAction {
+    return MDCActionSheetAction(title: "Email",
+                                image: UIImage(named: "Email")!) { (_) in
+                                  print("Email action") }
+  }
+
+  static var messageString: String {
+    return "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Curabitur ultricies diam " +
+      "libero, eget porta arcu feugiat sit amet. Maecenas placerat felis sed risusnmaximus tempus. " +
+      "Integer feugiat, augue in pellentesque dictum, justo erat ultricies leo, quis eleifend nisi " +
+    "eros dictum mi. In finibus vulputate eros, in luctus diam auctor in."
+  }
+
+  static func typical() -> MDCActionSheetController {
+    let actionSheet = MDCActionSheetController()
+    actionSheet.addAction(actionOne)
+    actionSheet.addAction(actionTwo)
+    actionSheet.addAction(actionThree)
+    return actionSheet
+  }
+
+  static func title() -> MDCActionSheetController {
+    let actionSheet: MDCActionSheetController = MDCActionSheetController(title: "Action Sheet")
+    actionSheet.addAction(actionOne)
+    actionSheet.addAction(actionTwo)
+    actionSheet.addAction(actionThree)
+    return actionSheet
+  }
+
+  static func message() -> MDCActionSheetController {
+    let actionSheet = MDCActionSheetController(title: nil,
+                                               message: messageString)
+    actionSheet.addAction(actionOne)
+    actionSheet.addAction(actionTwo)
+    actionSheet.addAction(actionThree)
+    return actionSheet
+  }
+
+  static func titleAndMessage() -> MDCActionSheetController {
+    let actionSheet = MDCActionSheetController(title: "Action Sheet",
+                                               message: messageString)
+    actionSheet.addAction(actionOne)
+    actionSheet.addAction(actionTwo)
+    actionSheet.addAction(actionThree)
+    return actionSheet
+  }
```

Comment:
I love this approach to breaking the individual examples out into named functions! Nice work here.

Reply:
Done